src/etcd/etcd_utils.h

```diff
@@ -0,0 +1,20 @@
+
+#ifndef CREDIS_UTILS_H
+#define CREDIS_UTILS_H
+
+#include <vector>
+#include "etcd3/include/etcd3.h"
+
+namespace utils {
+  class EtcdURL {
+   public:
+    std::string url;
+    std::string address; // host:port
+    std::string host;
+    int port;
+    std::string chain_prefix;
+  };
+  EtcdURL split_etcd_url(std::string url);
+};
+
+#endif //CREDIS_UTILS_H
```

Comment:
There needs to be two spaces after `endif` and one space after `//`.  These things can be done automatically by `clang-format`.